### ISSUE TITLE
Document methods in the wiki

### ISSUE THREAD

**Issue by Noswad-Zip** [ClarkThyLord/Voxel-Core#44]
Hi!

I believe that it could be helpful if class methods were documented in this project's documentation. The current documentation presents code examples, but those don't show how every method is used.

Of course, you could look at the source of the class you're trying to use. But as a user that is not too experienced with gdscript, that is a bit challenging.

Thank you.

**Comment by ClarkThyLord** [comment]
Yes, I had previously documented the code, but the constant changes in the rework made it tiresome and hard to maintain. I'll definitely do it now that the rework is stable though 👍
